libs/styles/styles.css

```diff
@@ -10,6 +10,10 @@
   --feds-totalheight-nav: calc(var(--feds-height-nav, --global-height-nav) + var(--feds-height-breadcrumbs, --global-height-breadcrumbs));
   --feds-localnav-height: 40px;
 
+  /* Global Z-Index Variables */
+  --modal-z-index: 102;
+  --modal-z-index-curtain: 101;
```

Comment:
could you also pull the other z-index values here as variables, which are defined further down in the styles.css?